mongodb/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM gitpod/workspace-full:latest
+
+# Install mongodb
+RUN sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 9DA31620334BD75D9DCB49F368818C72E52529D4
+RUN echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.0.list

```

Comment:
I see that your `deb` line mentions `bionic`, however our images are based on `cosmic`. I hope that's not an issue.